common/src/main/resources/assets/ftbquests/lang/fr_fr.json

```diff
@@ -0,0 +1,482 @@
+	"ftbquests.chapter.hide_quest_until_deps_visible": "Masquer les quêtes tant que les dépendances ne sont pas visibles",
+	"ftbquests.quest": "Quête",
+	"ftbquests.quests": "Quêtes",
+	"ftbquests.quest.completed": "Quête complétée !",
+	"ftbquests.quest.subtitle": "Sous-titre",
+	"ftbquests.quest.description": "Description",
+	"ftbquests.quest.x": "X",
+	"ftbquests.quest.y": "Y",
+	"ftbquests.quest.size": "Taille",
+	"ftbquests.quest.hide": "Masquer la quête tant que les dépendances ne sont pas visibles",
+	"ftbquests.quest.shape": "Forme",
+	"ftbquests.quest_link.shape": "Forme",
+	"ftbquests.quest_link.size": "Taille",
+	"ftbquests.quest_link.x": "X",
+	"ftbquests.quest_link.y": "Y",
+	"ftbquests.quest.shape.default": "Défaut",
+	"ftbquests.quest.shape.circle": "Cercle",
+	"ftbquests.quest.shape.square": "Carré",
+	"ftbquests.quest.shape.diamond": "Diamant",
+	"ftbquests.quest.shape.rsquare": "Carré arrondi",
+	"ftbquests.quest.shape.pentagon": "Pentagone",
+	"ftbquests.quest.shape.hexagon": "Hexagone",
+	"ftbquests.quest.shape.octagon": "Octogone",
+	"ftbquests.quest.shape.heart": "Cœur",
+	"ftbquests.quest.shape.gear": "Roue dentée",
+	"ftbquests.quest.can_repeat": "Quête répétable",
+	"ftbquests.quest.tasks_ignore_dependencies": "La tâche ignore les dépendances",
+	"ftbquests.quest.tasks_ignore_dependencies.tooltip": "Autoriser le commencement des tâches sans compléter les dépendances",
+	"ftbquests.quest.dependency_requirement": "Dépendances requises",
+	"ftbquests.quest.dependency_requirement.tooltip": "Ignorer si le nombre minimum de dépendances requises > 0",
+	"ftbquests.quest.dependency_requirement.all_completed": "Toute complétées",
+	"ftbquests.quest.dependency_requirement.one_completed": "Une complétée",
+	"ftbquests.quest.dependency_requirement.all_started": "Toutes commencées",
+	"ftbquests.quest.dependency_requirement.one_started": "Une commencée",
+	"ftbquests.quest.guide_page": "Pas de guide",
+	"ftbquests.quest.hide_dependency_lines": "Masquer les lignes de dépendances",
+	"ftbquests.quest.hide_dependency_lines.tooltip": "Contrôler l'affichage des lignes de dépendances des quêtes qui débloquent celle-ci",
+	"ftbquests.quest.hide_dependent_lines": "Masquer les lignes dépendantes",
+	"ftbquests.quest.hide_dependent_lines.tooltip": "Contrôler l'affichage des lignes pour les quêtes qui sont débloquées par celle-ci",
+	"ftbquests.quest.min_required_dependencies": "Nombre minimum de dépendances requises",
+	"ftbquests.quest.min_required_dependencies.tooltip": "Si vous mettez cette valeur à quelque chose > 0, elle devient une quête OU, pour laquelle il faut seulement compléter un certain nombre de dépendances pour la débloquer",
+	"ftbquests.quest.hide_text_until_complete": "Masquer le texte tant que la quête n'est pas complétée",
+	"ftbquests.quest.disable_jei": "Désactiver les recettes JEI",
+	"ftbquests.quest.optional": "Facultative",
+	"ftbquests.quest.locked": "Bloquée: Dépendances non complétées",

```

Comment:
In French there is a space between the word "bloquée" and the punctuation mark ":"